lib/config/definitions.ts

```diff
@@ -1486,6 +1492,24 @@ const options: RenovateOptions[] = [
     env: false,
     mergeable: true,
   },
+  {
+    name: 'rollupName',
+    description: 'Rollup name.',
+    type: 'string',
+    default: null,
```

Comment:
I think we wanted to move away from using `null` as a default value?

Reply:
`null` is definitely a bad idea instead of `true/false` although I think it's OK for strings. We could use an empty string instead.

Reply:
Do we need a default? Can't it simply be undefined for string?

Reply:
I'll let you decide what you like best. 😉